src/pages/iou/IOUModal.js

```diff
@@ -247,35 +263,50 @@ class IOUModal extends Component {
         if (this.state.currentStepIndex >= this.steps.length - 1) {
             return;
         }
+
         this.setState(prevState => ({
             previousStepIndex: prevState.currentStepIndex,
             currentStepIndex: prevState.currentStepIndex + 1,
         }));
     }
 
     /**
-     * Update comment whenever user enters any new text
+     * Checks if user has a GOLD wallet, then creates a paid IOU report on the fly
      *
-     * @param {String} comment
+     * @param {String} paymentMethod
      */
-    updateComment(comment) {
-        this.setState({
-            comment,
-        });
+    sendMoney(paymentMethod) {
+        const hasGoldWallet = this.props.userWallet.tierName && this.props.userWallet.tierName === CONST.WALLET.TIER_NAME.GOLD;
+
+        // If the user is trying to send money, then they need to upgrade to a GOLD wallet
+        if (this.isSendRequest && !hasGoldWallet) {
```

Comment:
Do we still need `this.isSendRequest`? We are sending money so it must be `isSendRequest`

Reply:
Oh yup, can get rid of this

Reply:
Ah, I think you fixed some other thing and not the thing I was hinting at 😄